packages/sakuli-legacy/src/context/sahi/accessor/accessor-util.class.ts

```diff
@@ -253,11 +253,17 @@ export class AccessorUtil {
     return identifier.startsWith("/") && identifier.endsWith("/");
   }
 
+  /**
+   * Allows the user to use an index at the end of the string, e.g.
+   *
+   * _password("shaded bigfont[1]")
+   * This matches the second password field with the css class 'shaded bigfont'.
+   */
   async getByString(
     elements: WebElement[],
     identifier: string
   ): Promise<WebElement[]> {
-    const indexRegExp = /.*\[([0-9]+)\]$/;
+    const indexRegExp = /.*\[([0-9]+)]$/;
```

Comment:
IntelliJ suggested it because it's a redundant escape as there is no matching opening bracket for a character set.

Reply:
Hmm, ok.

I was just asking because even if it's redundant, it's way easier for me to process that this should explicitly match `]` and does not belong to a character set.